src/RabbitMQAttribute.cs

```diff
@@ -47,6 +47,13 @@ public sealed class RabbitMQAttribute : Attribute
         /// </summary>
         public int Port { get; set; }
 
+        /// <summary>
+        /// Gets or sets the vhost to use when connecting to RabbitMQ using HostName / UserName / Password
+        /// <see href="https://www.rabbitmq.com/vhosts.html"/>.
+        /// </summary>
+        [AppSetting]

```

Comment:
Delete line 54. `AppSetting` attribute will require the value for `VirtualHost` parameter to always come from settings file. We add that restriction only for parameters who values might be secrets.